sqlmesh/core/context.py

```diff
@@ -2607,6 +2609,7 @@ def _context_diff(
         force_no_diff: bool = False,
         ensure_finalized_snapshots: bool = False,
         diff_rendered: bool = False,
+        always_init_from_prod: bool = False,
```

Comment:
why do we need this argument? can't we access the config in this method directly?